### PR TITLE
2 home page

### PR DESCRIPTION
**Pull request by chansiky**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/components/category/popular-category.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  constructor(props){
+    super(props)
+  }
+
+  async componentDidMount(){
+    await this.props.getPopularCategories()
+  }
+
+  render() {
+    console.log('props inside popularCategories is: ', this.props)
```
Comment (jamigibbs) [inline client/components/category/popular-category.js]: Console log should be removed.

File: client/components/paths/popular-paths.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { getPopularPathsInAllCategories } from '../../store'
+import { PathCard } from './index'
+import Grid from '@material-ui/core/Grid'
+
+class PopularPaths extends React.Component {
+  constructor(props){
+    super(props)
+  }
+
+  async componentDidMount(){
```
Comment (jamigibbs) [inline client/components/paths/popular-paths.js]: You probably don't need to do async/await on componentDidMount?

File: client/components/paths/popular-paths.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    await this.props.getPopularPaths()
+  }
+
+  render() {
+    console.log('props inside popularPaths is: ', this.props)
```
Comment (jamigibbs) [inline client/components/paths/popular-paths.js]: Console log can be removed.

File: server/api/categories.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ let session = driver.session();
 const router = require('express').Router()
 //dummy code
 const dummyData = require('../../script/data/path.js')
+const dummyCategories = require('../../script/data/category.js')
```
Comment (jamigibbs) [inline server/api/categories.js]: Maybe don't need dummy content anymore?

File: server/api/paths.js [hunk 5]
```diff
@@ -2,6 +2,7 @@ let neo4j = require('neo4j-driver').v1;
 let driver = neo4j.driver("bolt://localhost", neo4j.auth.basic("neo4j", "1234"))
 let session = driver.session();
 const router = require('express').Router()
+const dummyPaths = require('../../script/data/path.js')
```
Comment (jamigibbs) [inline server/api/paths.js]: More dummy content. Not sure if this is needed anymore.

File: server/api/categories.js (hunk 4 above)
Comment (maubertw) [reply server/api/categories.js]: true dat

File: server/api/paths.js (hunk 5 above)
Comment (maubertw) [reply server/api/paths.js]: same

**Pull request merged**